iomgr/lib/dbengine/parser/AntlrHelpers.h

```diff
@@ -14,6 +14,14 @@ constexpr std::size_t kInvalidNodeType = std::numeric_limits<std::size_t>::max()
 
 namespace helpers {
 
+/**
+ * Converts inplace two single quote into one single quote, as per SQL string literal syntax.
+ * NOTE: in SQL, a single quote is escaped with a single quote.
+ * @param s A string.
+ * @return Same string.
+ */
+std::string& fixSingleQuotes(std::string& s);
+
 /**
  * Removes leading and trailing quotes from string.
```

Comment:
Since this change affects functionality of these functions, their descriptions must be updated, otherwise documentation misrepresents actual functionality. I suggest to add in the end: `, fixes single quotes as per SQL string literal syntax`